src/lang/triage-meeting-procedure.md

```diff
@@ -0,0 +1,61 @@
+# Triage meeting procedure
+
+This page documents how to run a lang team triage meeting,
+should you have the misfortune of being forced to do so.
+
+## Attending a meeting
+
+If you would just like to **attend** a lang-team triage meeting, all
+you have to do is join the zoom call (the URL is attached to the
```

Comment:
Link it here too tho for convenience?

Reply:
I did not put it here because I was afraid it might change and I wanted "one source of truth"